src/main/java/nl/praegus/fitnesse/slim/fixtures/mockserver/MockServer.java

```diff
@@ -265,6 +267,13 @@ private HttpRequest httpRequestMatching(Map<String, Object> rules) {
                         req = req.withHeader(header.getKey().toString(), header.getValue().toString());
                     }
                     break;
+                case "body":
+                    if (rules.get(rule).toString().matches("^\\$.*$")) {
```

Comment:
Ik zou hier gewoon `.startsWith("$")` gebruiken i.p.v. `.matches(regex)`? Is wat leesbaarder en naar ik verwacht efficienter?

Reply:
Je hebt helemaal gelijk. Ga ik aanpassen.